cisagov/manage.get.gov
# Issue # 1106 - fix "required field" flag on public contact fields

## Ticket 1106

Resolves #1106


## Changes


- Added blank=True to fields in public-contact so that the frontend form will allow blanks for nullable fields instead of (incorrectly) marking those fields as required.



## Context for reviewers



## Setup



## Code Review Verification Steps
1. go to django admin and click PublicContact
2. click one of the PublicContact objects
3. click save
4. Verify that nullable fields (marked as nullable in the frontend) don't raise a "required" flag

### As the original developer, I have

#### Satisfied acceptance criteria and met development standards


#### Ensured code standards are met (Original Developer)


#### Validated user-facing changes (if applicable)


### As a code reviewer, I have

#### Reviewed, tested, and left feedback about the changes


#### Ensured code standards are met (Code reviewer)


#### Validated user-facing changes as a developer




**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

### As a designer reviewer, I have

#### Verified that the changes match the design intention


#### Validated user-facing changes as a designer




## Screenshots
